### DISCUSSION TITLE
cs/unicode/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#21] [General]
# cs/unicode/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/unicode/

**Comment by RaineyZheng** [comment]
关于BOM的地方是不是写错了

> UTF-16 可以使用大端序和小端序，在文件开头会有 BOM（U+FEFF）来指明到底是哪个模式（如果没有，则需要猜测）
> 小端序（UTF-16 LE）下 BOM 表现为 **0xFF 0xEF**
> 大端序（UTF-16 BE）下 BOM 表现为 **0xEF 0xFF**

BE应该是0xFE 0xFF，LE应该是0xFF 0xFE

**Reply by TonyCrane** [reply]
草，EF 写反了
